cypress/e2e/GUI/gui.cy.js

```diff
@@ -0,0 +1,104 @@
+/// <reference types="Cypress"/>
+
+describe('Validar Pagina inicial', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:3000')
+    })
+
+    it('Valida se H1 esta visivel e alterna tema', () => {
```

Comment:
```suggestion
    it('Valida se h1 esta visivel e alterna tema', () => {
```

Use a tag como no HTML, ou seja, com letra minúscula.